bijection-scrooge/src/main/scala/com/twitter/bijection/scrooge/ScroogeCodecs.scala

```diff
@@ -35,10 +35,19 @@ object BinaryScalaCodec {
 }
 
 class BinaryScalaCodec[T <: ThriftStruct](c: ThriftStructCodec[T])
```

Comment:
Probably, it'll take more work to go through its serialization to ensure its sane though. Do we know if its used much to make it worth it?